Makefile

```diff
@@ -387,9 +406,11 @@ dist_export:
 	@mkdir -p $(DIST_EXPORT_DIR)
 	# Do a basic copy of the add-on.
 	$(RSYNC) $(BUILT)/$(ADD-ON)/* $(DIST_EXPORT_DIR)
-	# Use the install.rdf.in rather than install.rdf.
+	# Use a modified install.rdf.in rather than install.rdf.
 	rm -f $(DIST_EXPORT_DIR)/install.rdf
-	$(RSYNC) $(ADD-ON)/install.rdf.in $(DIST_EXPORT_DIR)
+	# mozilla-* doesn't need the date field.
```

Comment:
mozilla-* is a fairly "insider" term.  Maybe "mozilla-central and other firefox repositories"?